app/serializers/api/state_serializer.rb

```diff
@@ -1,5 +1,7 @@
 class Api::StateSerializer < ActiveModel::Serializer
-  attributes :id, :name, :abbr
+  attributes :id, :name, :abbr, :display_name
+
+  delegate :display_name, to: :object
```

Comment:
I don't think this line is necessary. Any methods available on the object are serialisable as attributes. They don't need to be attributes in the sense of ActiveRecord.